Uppsala/UppQ1524o.xml

```diff
@@ -0,0 +1,566 @@
+                                <title type="complete" ref="LIT7099QeneEffoHalloka"/>
+                                <incipit xml:lang="gez">
+                                    <choice><sic>አ</sic><corr resp="JK">እ</corr></choice>ፎ፡ ሀሎከ፡ በጊዜ፡ 
+                                    ትቤ<del rend="overUnderlined strikethrough">ሳ</del><add place="above">ላ</add>፡ ኦሪተ፡ ሙሴ፡ ቅዱስ፤
+                                </incipit>
+                                <note>In the upper left corner of the folio, the following note has been written: 
+                                    "G.L."</note>
+                            </msItem>
+                            
+                            
+                            <msItem xml:id="p1_i22">
+                                <locus target="#45r #46r"/>
+                                <title type="complete" ref="LIT7101QeneAmh"/>
+                                <incipit xml:lang="am">
+                                    <choice><sic>ኢ</sic><corr resp="HSM">እየ</corr></choice>ደ<choice><sic>ርስበ</sic><corr resp="HSM">ረሰብ</corr></choice>ኝ፡ 
+                                    ነው፡ በሰኔ፡ ፈረሰኛው፡ ሞት፡ ነፍሴን፡ ሳልጠቅማት፡ ወየው፡ ሰዎች፡ 
+                                    <choice><sic>ኢ</sic><corr resp="HSM">እ</corr></choice>ያሉ፡ አዝማሪ፤
+                                </incipit>
+                                <note>In the upper left corner of the folio, the following note has been written: 
+                                    "G.L." In the upper right corner (under the <foreign xml:lang="gez">qəne</foreign> number),
+                                there is a note in Swedish: "<foreign xml:lang="sv">amhar</foreign>" ("Amharic"). Under this,
+                                another hand has written the following note in pencil in Tigrinya: 
+                                    <foreign xml:lang="">ካብ፡ ዋጣ፡ ከም፡ ስር፡ ዓት፡ ቅነ፡ እተገብረ።</foreign>
+                                    ("Which was made by a <foreign xml:lang="ti">wāṭā</foreign> according 
+                                    to the order of <foreign xml:lang="ti">qəna</foreign>").</note>
+                            </msItem>
+                            
+                            <msItem xml:id="p1_i23">
+                                <locus target="#47r"/>
+                                <title type="complete" ref="LIT7100QeneLeyon"/>
+                                <incipit xml:lang="gez" type="inscriptio">
+                                    ዋዜማ፡ ቅኔ።
+                                </incipit>
+                                <incipit xml:lang="gez">
+                                    ልዮን፡ ወአውጣኪ፡ ምስለ፡ ድዮስቆሮስ፡ ሊቅ፡ በባሕለ፡ እሙን፡ ኢሐብሩ።
+                                </incipit>
+                                <note>In the upper left corner of the folio, the following note has been written: 
+                                    "Twoldo".</note>
+                            </msItem>
+                            
+                        </msContents>
```

Comment:
I think some minimal physDesc for each msPart is desirable to make the composition clear